health-services/project-factory/src/server/validators/campaignValidators.ts

```
@@ -1046,7 +1046,7 @@ async function validateCampaignName(request: any, actionInUrl: any) {
 }
 
 async function validateById(request: any) {
-    const { id, tenantId, action } = request?.body?.CampaignDetails
+    const { id, tenantId } = request?.body?.CampaignDetails
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)